store/tooling/tooling.go

```diff
@@ -320,6 +323,7 @@ func (tsto *ToolingStore) DownloadMany(toDownload []SnapToDownload, curSnaps []*
 			TrackingChannel:  ch,
 			Epoch:            csnap.Epoch,
 			IgnoreValidation: !opts.EnforceValidation,
+			ValidationSets:   opts.ValidationSets,
```

Comment:
if this method has tests they should be extended about this?

Reply:
DownloadMany is untested in tooling_test.go, however I can do a PR about some tests for it after this chain